### PR TITLE
Modified to use ghcr.io as a container registry

### LINKED ISSUE #455: Use ghcr.io for conatiner registry
## What

We now use quay.io for container registry, but it is boring to manage users and credentials.
Github provides a container registry service. It is free for public images and its access control is integrated with GitHub.

## How

- enable github container registry
- migrate all images from quay.io to ghcr.io
- rewrite image names on codes.

## Checklist

- [ ] Finish implementation of the issue
- [ ] Test all functions
- [ ] Have enough logs to trace activities
- [ ] Notify developers of necessary actions

### PR DESCRIPTION
**Pull request by bells17**
fix #455

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Documentation
 Docker images
 -------------
 
-Docker images are available on [Quay.io](https://quay.io/organization/topolvm)
+Docker images are available on [ghcr.io](https://github.com/orgs/topolvm/packages)
```
Comment (toshipp) [inline README.md:79]: Please add a note that images on quay.io are now deprecated and will be unavailable in the future.

File: README.md [hunk 2]
```diff
@@ -76,7 +76,8 @@ Documentation
 Docker images
 -------------
 
-Docker images are available on [Quay.io](https://quay.io/organization/topolvm)
+Docker images are available on [ghcr.io](https://github.com/orgs/topolvm/packages)  
+**Please note that [the images on quay.io](https://quay.io/organization/topolvm) is currently deprecated and will no longer be available in the future.**
```
Comment (toshipp) [inline README.md:80]:
```suggestion
**Please note that [the images on quay.io](https://quay.io/organization/topolvm) are currently deprecated and will no longer be available in the future.**
```

**Review by toshipp — CHANGES_REQUESTED** [review]

**Review by toshipp — APPROVED** [review]

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -19,10 +19,10 @@ jobs:
           if [ $(echo $VERSION | grep "-") ]; then PRERELEASE=true; else PRERELEASE=false; fi
           echo "::set-output name=version::${VERSION}"
           echo "::set-output name=prerelease::${PRERELEASE}"
-      - run: echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USER }} --password-stdin quay.io
+      - run: echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u ${{ secrets.GHCR_USER }} --password-stdin
```
Comment (llamerada-jp) [inline .github/workflows/release.yaml:22]: memo:
We can login to ghcr using GITHUB_TOKEN according to the following document.
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry

**Review by llamerada-jp — APPROVED** [review]
LGTM

**Pull request merged**